component/discovery/process/discover.go

```diff
@@ -118,7 +118,6 @@ func discover(l log.Logger, cfg *DiscoverConfig) ([]process, error) {
 			username, err = p.Username()
 			if err != nil {
 				loge(int(p.Pid), err)
-				continue
```

Comment:
Does this mean that a `__meta_process_username` label will still be added, but with a value of `""`?

Reply:
No the convertProcess has filters for the null values

https://github.com/grafana/agent/blob/1f3943dddbf9f1b8a4146bcd95f88be8aadbfa38/component/discovery/process/discover.go#L67